### PR TITLE
BOOKKEEPER-1017 Create documentation for ZooKeeper ACLs

### PR DESCRIPTION
**Pull request by eolivelli**
This is the documentation for ZooKeeper security and there is an intro about security in general.
It is work-in-progress, I created this PR in order to make it visible and gather suggestions while writing

### CODE REVIEW

**Comment by sijie** [comment]
@eolivelli if it is WIP, it is good to add change the PR title to be "WIP - XXXX"

File: doc/bookkeeperSecurity.textile [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        .
+        .
+
+h1. Security in BookKeeper
+
+Apache BookKeeper is a decentralized platform and there are multiple aspects to deal with while securing an BookKeeper Cluster.
+
+Metadata are stored on ZooKeeper, so first of all you will need to secure your ZooKeeper cluster, see "ZooKeeper security":https://zookeeper.apache.org/security.html
+
+Then you have to take care of access to Bookies, you can configure Authentication and encryption using TLS, out of the box BookKeeper supports Kerberos authentication, DIGEST-MD5 authentication and TLS encryption. You can also leverage TLS client authentication in order to protect your data.
```
Comment (jiazhai) [inline doc/bookkeeperSecurity.textile]: In “Authentication and encryption”, one is capitalised, while the other is not, Is there a need to unify them here? 
And is there a need to add a link for Kerberos here?

Comment (jiazhai) [inline doc/bookkeeperSecurity.textile]: In “Authentication and encryption”, one is capitalised, while the other is not, Is there a need to unify them here?
And is there a need to add a link for Kerberos here?

**Comment by eolivelli** [comment]
@jiazhai @sijie I have finished my proposal. It is ready for review.

I will start writing about BookKeeper Authentication soon

**Comment by eolivelli** [comment]
@jiazhai  is this OK for you now ?

**Pull request closed without merging**